src/main/java/org/opentripplanner/netex/loader/parser/NetexDocumentParser.java

```diff
@@ -41,6 +41,10 @@ public static void parseAndPopulateIndex(NetexEntityIndex index, PublicationDeli
         new NetexDocumentParser(index).parse(doc);
     }
 
+    public static void finnishUp() {
```

Comment:
FYI Typo

Reply:
Maybe he meant that all the Finnish people should stand up? :)